charts/sysdig-deploy/README.md

```diff
@@ -48,48 +48,48 @@ Currently included components:
 
 4. Do one of the following:
 
-   - Using the release name `sysdig`, run the following snippet to install the release into the namespace `sysdig-agent`:
+    - Using the release name `sysdig`, run the following snippet to install the release into the namespace `sysdig-agent`:
 
-    ```bash
-   helm install sysdig sysdig/sysdig-deploy \
-       --namespace sysdig-agent \
-       --set global.sysdig.accessKey=ACCESS_KEY \
-       --set global.sysdig.region=SAAS_REGION \
-       --set global.clusterConfig.name=CLUSTER_NAME
-    ```
+      ```bash
+      helm install sysdig sysdig/sysdig-deploy \
+          --namespace sysdig-agent \
+          --set global.sysdig.accessKey=ACCESS_KEY \
+          --set global.sysdig.region=SAAS_REGION \
+          --set global.clusterConfig.name=CLUSTER_NAME
+        ```
 
-     **GKE Autopilot**: GKE Autopilot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install the Sysdig agent:
+      **GKE Autopilot**: GKE Autopilot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install the Sysdig agent:
 
       ```bash
-   helm install sysdig sysdig/sysdig-deploy \
-        --namespace sysdig-agent \
-        --set global.sysdig.accessKey=ACCESS_KEY \
-        --set global.sysdig.region=SAAS_REGION \
-        --set global.clusterConfig.name=CLUSTER_NAME \
-        --set agent.gke.autopilot=true \
-        --set agent.slim.enabled=false \
-        --set nodeAnalyzer.enabled=false
+      helm install sysdig sysdig/sysdig-deploy \
+          --namespace sysdig-agent \
+          --set global.sysdig.accessKey=ACCESS_KEY \
+          --set global.sysdig.region=SAAS_REGION \
+          --set global.clusterConfig.name=CLUSTER_NAME \
+          --set agent.gke.autopilot=true \
+          --set agent.slim.enabled=false \
+          --set nodeAnalyzer.enabled=false
       ```
 
 
-   - Install with a values file. 
+    - Install with a values file. 
 
-     To do so, create a new file `values.sysdig.yaml`:
+      To do so, create a new file `values.sysdig.yaml`:
 
-        ```yaml
-     global:
-       sysdig:
-         accessKey: ACCESS_KEY
-         region: SAAS_REGION
-       clusterConfig:
-         name: CLUSTER_NAME
-        ```
+       ```yaml
+       global:
+         sysdig:
+           accessKey: ACCESS_KEY
+           region: SAAS_REGION
+         clusterConfig:
+           name: CLUSTER_NAME
```

Comment:
good catch! but how is this going to fix the rendering problem, and is there a way of testing this branch?

Reply:
Here's the GH Pages render on my fork: https://patrickeasters.github.io/charts/charts/sysdig-deploy/
Here's the README showing the code blocks properly indent now: https://github.com/sysdiglabs/charts/blob/0d96369d11458435336db4b6989bdf2d55d75800/charts/sysdig-deploy/README.md

Reply:
nice. lookin good thank you